### ISSUE TITLE
synology NAS upgrade after Traccar service not run

### ISSUE THREAD

**Issue by aydnykn** [traccar/traccar#2555]
yesterday  I'm upgrade DS216+II (6.0.2-8451update 3 to 4) and restart machine.
After start machine Traccar service stopped.
I started the Traccar service manually, after a 15 seconds the Traccar stopped again.
It seems that the CPU is running 100% during that short run.

uninstall traccar and drop traccar database, 
uninstall all java, restart NAS, install Java and traccar but traccar work 10 seconds and stop servise, 
manuel start traccar service again stop. 
I dont understant what a problem?

**Comment by CVSz** [comment]
check on logs/tracker-server.log and logs/wrapper.log

**Comment by aydnykn** [comment]
Here is the log file, problem is still going on.
[tracker-server.log.20161118.txt](https://github.com/tananaev/traccar/files/603056/tracker-server.log.20161118.txt)
[tracker-server.txt](https://github.com/tananaev/traccar/files/603057/tracker-server.txt)
[wrapper.log.20161119.txt](https://github.com/tananaev/traccar/files/603058/wrapper.log.20161119.txt)

**Comment by tananaev** [comment]
Looks like you have some problem with the database. I guess you have some non-default schema, so it fails to migrate. Other possibility is that you have an ancient version of MySQL.

**Comment by aydnykn** [comment]
Dear tananaev, 
What I did before your comment;
uninstall traccar and drop traccar database,
uninstall all java, uninstall phpmyadmin and restart Synology NAS, 
after install Java, traccar and phpmyadmin. but I did'nt get a result.

When you say "**ancient version of MySQL**" I'm remove and install Mariadb. 
and install traccar; Yes, Traccar is working right now.
Thank you for your interest, good luck
